src/utils/firebase.js

```diff
@@ -0,0 +1,21 @@
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+//Initialize Firebase
+if (firebase.apps.length === 0) {
+  firebase.initializeApp(firebaseConfig);
```

Comment:
I did not commit the config object on this branch, we need to find a way to manage the credentials